web/themes/custom/server_theme/templates/server-theme-person-card.html.twig

```diff
@@ -0,0 +1,53 @@
+<div
```

Comment:
Please follow Drupal's coding standards

Reply:
done

Reply:
No, you still have tabs :/

Reply:
tabs should be gone now, Twig formatter was converting spaces to tabs on save.